Makefile

```diff
@@ -44,6 +46,10 @@ $(protoc_gen_typescript_http):
 	$(info [$@] building protoc-gen-typescript-http...)
 	@go build -o $@ .
 
+.PHONY: $(eslint)
+eslint: $(eslint)
+	$(info [$@] linting typescript files...)
+	$(eslint) --config $(eslint_cwd)/.eslintrc.js --quiet "examples/proto/gen/typescript/einride/example/freight/v1/index.ts"
```

Comment:
Can we run eslint on the entire generated typescript dir instead?

Reply:
Something like `"examples/proto/gen/typescript/**/*.ts"`

Reply:
Yeah good point! will update 👍

Reply:
Updated and fixed all lint issues! There was a few edge cases with enums (e.g no leading pipe if only 1 enum or ending the last enum with a semi colon). I'm not sure my solution is the best but it works 🙈